lib/mi_bridges/driver/job_income_information_page.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module MiBridges
+  class Driver
+    class JobIncomeInformationPage < BasePage
+      delegate :primary_member, to: :snap_application
```

Comment:
seems like we are doing this delegation and the `first_name` delegation in many steps. Thoughts on putting them in a shared place somewhere?

Reply:
Yup! For sure. It's common enough that it should be there for all the steps. Maybe prefixed as well - `primary_member_first_name`